AppCenterAuth/AppCenterAuth/MSAuth.m

```diff
@@ -262,6 +264,23 @@ - (void)signOut {
 
 #pragma mark - Private methods
 
+- (void)enableMSALLogging {
+  MSALGlobalConfig.loggerConfig.logLevel = MSALLogLevelVerbose;
+  [MSALGlobalConfig.loggerConfig setLogCallback:^(MSALLogLevel level, NSString *message, BOOL containsPII) {
+    if (!containsPII) {
```

Comment:
are we missing something important? if yes any way to obfuscate?

Reply:
Basically we don't know what should be obfuscate. We can figure this out but I think this would be good for the first step.